src/System.Collections/tests/Generic/Comparers/Comparer.Generic.Tests.cs

```diff
@@ -32,24 +32,27 @@ public void Comparer_EqualsShouldBeOverriddenAndWorkForDifferentInstances()
             Assert.False(comparer.Equals(3));
             Assert.False(comparer.Equals("foo"));
             Assert.False(comparer.Equals(Comparer<Task<T>>.Default));
+        }
 
+        [Fact]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetcoreUwp, "")]
+        public void Comparer_EqualsShouldBeOverriddenAndWorkForDifferentInstances_cloned()
+        {
             // If we are running on full framework/CoreCLR, Comparer<T> additionally
             // overrides the Equals(object) and GetHashCode() methods for itself,
             // presumably to support serialization, so test that behavior as well.
             // This is not done in .NET Native yet: dotnet/corert#1736
```

Comment:
Delete the lengthy comment as it will be captured in the SkipTargetFramework's second arg.